initialize_language.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+if [ $# -ne 1 ]
+then
+	echo "usage: $0 language_code"
+	exit 0
+fi
+
+IROHA_LANG="$1"
+IROHA_LANG_DIR="$PWD/docs/locale/$IROHA_LANG"
+IROHA_DOCS_DIR="$IROHA_LANG_DIR/docs"
+
+# Prepare folder for translation
+mkdir -p "$IROHA_DOCS_DIR"
+
+# Create symlinks for docs directory
+lndir "$PWD"/iroha/docs/ "$IROHA_DOCS_DIR"
+
+# Replace absolute symlinks with relative
+symlinks -rc "$IROHA_LANG_DIR"
+
+# Replace symlinks with file copies for rst, md, csv, yaml
+for f in $(find "$IROHA_LANG_DIR" -type l \( -name '*.rst' -o -name '*.md' -o -name '*.csv' -o -name '*.yaml' \))
+do
+	cp --remove-destination $(readlink -f $f) $f
+done
+
+# Copy CONTRIBUTING.md from iroha repo root
+cp "$PWD"/iroha/CONTRIBUTING.md "$IROHA_LANG_DIR"/
```

Comment:
```suggestion
# Copy CONTRIBUTING.* from iroha repo root
cp "$PWD"/iroha/CONTRIBUTING.* "$IROHA_LANG_DIR"/
```
Probably the easiest way now would be to copy this file by wildcard, because it is still an exception to other files, as it is not stored in docs directory. It might be better to move it to docs and leave a symlink here, or vice versa.

Another option would have been to copy all the rst and md files from the root directory, but we have lots of empty md files which are currently not used in the documentation. Maybe we should get back to the task of filling those md files, and using them in the documentation as well?